.vscode/settings.json

```diff
@@ -1,5 +1,9 @@
 {
     "C_Cpp.inactiveRegionOpacity": 0.55,
     "C_Cpp.errorSquiggles": "enabled",
-    "C_Cpp.default.compilerPath": "C:\\msys64\\ucrt64\\bin\\gcc.exe"
+    "C_Cpp.default.compilerPath": "C:\\msys64\\ucrt64\\bin\\gcc.exe",
+    "files.associations": {
+        "*.embeddedhtml": "html",
+        "typeinfo": "cpp"
+    }
```

Comment:
In the next commit, add this to the `.gitignore` file please

Reply:
All the `.vscode` folder 🙏

Reply:
Done!